twistlock/datadog_checks/twistlock/utils.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+NORMALIZATION_FIELDS = {
+    "complianceVulnerabilities": "complianceIssues",
+    "complianceVulnerabilitiesCnt": "complianceIssuesCount",
+    "cveVulnerabilities": "vulnerabilities",
+    "cveVulnerabilitiesCnt": "vulnerabilitiesCount",
+    "cveVulnerabilityDistribution": "vulnerabilityDistribution",
+    "pkgDistro": "osDistro",
+    "pkgDistroRelease": "osDistroRelease",
+}
+
+
+def normalize_api_data_inplace(data):
+    """
+    Normalize api data to make it compatible with both standalone Twistlock and Prisma Cloud Twistlock.
+    Normalization based on https://docs.paloaltonetworks.com/prisma/prisma-cloud/19-11/prisma-cloud-compute-edition-admin/api/porting_guide.html
+    """  # noqa: E501
```

Comment:
note: `# noqa: E501` because of the long url

Reply:
If you put the URL on its own line, I *think* flake8 should not complain, does it?

```python
"""
...
Normalization based on:
https://docs.paloaltonetworks.com/prisma/prisma-cloud/19-11/prisma-cloud-compute-edition-admin/api/porting_guide.html
"""
```

Reply:
Right, didn't knew that. Changed.